.eslintrc.json

```diff
@@ -0,0 +1,43 @@
+{
+    "env": {
+        "browser": true,
+        "node": true,
+        "es2021": true
+    },
+    "plugins": [
+        "@stylistic"
+    ],
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "ignorePatterns": [
```

Comment:
I'm getting

```
Error: ESLint configuration in .eslintrc.json is invalid:
	- Unexpected top-level property "ignorePatterns".
```

I'm sure it's likely I just didn't configure eslint correctly

```
$ eslint --version
v6.4.0
```

Reply:
I'm using eslint `8.57.0`

actually, I already added needed eslint into `package.json`, you should be able to just `npm i` and then `npx eslint` to start using it, where did you even dig up that [ancient](https://eslint.org/blog/2019/09/eslint-v6.4.0-released/) eslint 6.4.0 from?

```
mbin % git checkout new/eslint-basic-integration
Switched to branch 'new/eslint-basic-integration'
mbin % npm i

added 98 packages, and audited 853 packages in 3s

133 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
mbin % npx eslint --version
v8.57.0
```

also eslint 9?? apparently released like 2 days ago???, should I upgrade this to eslint 9 too while I'm at it? looks like there's some config format and organizational changes so it needs a bit more change than just update the installed version

Reply:
I guess that's a problem I have, as `npm i` (i thought you had typo'd, didn't realize it was a real command, haha) gets me

```
npm ERR! enoent ENOENT: no such file or directory, open '/git/MbinOrg/mbin/vendor/symfony/stimulus-bundle/assets/package.json'
```

Reply:
I guess I'll probably be fighting with composer/symfony as neither want to install on my system for some reason. That's why I kind of always hope for container solutions or binaries but, that's going off topic, I guess I'll just mark this as resolved since it's a me problem